PlaceRouteHierFlow/PnR-pybind11.cpp

```diff
@@ -492,7 +492,9 @@ PYBIND11_MODULE(PnR, m) {
     .def_readwrite("max_init_trial_count", &PlacerHyperparameters::max_init_trial_count)
     .def_readwrite("max_cache_hit_count", &PlacerHyperparameters::max_cache_hit_count)
     .def_readwrite("select_in_ILP", &PlacerHyperparameters::select_in_ILP)
+    .def_readwrite("use_ILP_placer", &PlacerHyperparameters::use_ILP_placer)
```

Comment:
It would be nice to add comments to describe what select_in_ILP, use_ILP_placer and ilp_solver do.